go/logic/migrator.go

```diff
@@ -396,6 +396,10 @@ func (this *Migrator) handleCutOverResult(cutOverError error) (err error) {
 	// Only on error:
 
 	if this.migrationContext.TestOnReplica {
+		if err := this.hooksExecutor.onTestReplicaComplete(); err != nil {
```

Comment:
Is this the correct spot for it? I am aiming to have this run after replication has stopped, cutover is performed but before the replication is started again. I _think_ this is where it should be however I'd like some more 👀  in case I've buggered something up here.